app/views/member_mailer/welcome_student.html.haml

```diff
@@ -4,26 +4,18 @@
 
 %p
   - if @member.subscriptions.any?
-    You'll automatically receive email invitations to our workshops as soon as they are open for RSVP.
-    - if @member.chapters.pluck(:name).include?("London")
-      However, our London chapter doesn't send workshop invitations via email, so you won't receive email invites for regular London events. We open RSVPs at around 1pm on the Friday before the workshop, and you will be able to register through the workshop page on our website. To be notified when we open RSVPs, join the #london-chapters channel
-      = link_to 'on our Slack', 'http://slack.codebar.io'
-      or
-      = link_to 'follow us', 'https://twitter.com/codebarLDN'
-      on Twitter.
-  - else
-    We send out invitations to our events via email.
```

Comment:
@KimberleyCook I've updated the tests to pass, but was wondering, do we want to remove this link from the email? I think it would still be useful to have it.

Reply:
Yeah, let's not remove it.